builder/docker/javascript/Dockerfile

```diff
@@ -2,11 +2,11 @@ FROM ghcr.io/suborbital/javy:v0.3.0 as javy
 FROM suborbital/subo:dev as subo
 
 FROM node:16-bullseye-slim
-WORKDIR /root/runnable
+WORKDIR /usr/src/runnable
 # Propagate our root permissions for our home folder to everyone. This allows
 # npm scripts (which get run as whatever user owns the mounted runnable
 # directory) to access common home folder resources (caches, etc.).
-RUN mkdir /root/suborbital && chmod -R o=u /root
+# RUN mkdir /root/suborbital && chmod -R o=u /root
```

Comment:
```suggestion
```

Reply:
Thanks, I didn't want to touch it until we knew it was harmless to remove. :)